buildSrc/buildWebapp.js

```diff
@@ -32,7 +32,7 @@ export async function buildWebapp({ version, stage, host, measure, minify, proje
 	const polyfillBundle = await rollup({
 		input: ["src/polyfill.ts"],
 		plugins: [
-			typescript(),
+			typescript({ outputToFilesystem: true }),
```

Comment:
it is unclear why we do output to filesystem, maybe add a comment? (also I do not think it should be necessary?)